container-storage-setup.spec

```diff
@@ -26,21 +26,11 @@ as the root logical volume and partition table.
 %install
 %make_install
 
-%post
-%systemd_post %{name}.service
-
-%preun
-%systemd_preun %{name}.service
-
-%postun
-%systemd_postun %{name}.service
-
```

Comment:
Because container-storage-setup does not include services.  Those need to move to docker package.